File: assignment1/question4/array.cc
#include <iostream>
using namespace std;
int random(n)
{
    
}
int main()
{
srand((unsigned) time(0));
int n,c;
cout<<"Plaese enter the size of the array ";
cin>>n;
int a[n],b[20];
for (int i=0;i<n;i++)
{
a[i]=rand();
}
cout<<"Random numbers are "<<endl;
for(int i=0;i<n;i++)
{
cout<<a[i]<<" ";
}
cout<<"\n20 random numbers are "<<endl;
for(int i=0;i<20;i++)
{
b[i]=rand();
cout<<b[i]<<" ";
}
cout<<"\n20 random numbers in ascending order are "<<endl;
for(int j=0;j<20;j++)
{
for(int i=0;i+1<20-j;i++)
{
if(b[i]>b[i+1])
{
c=b[i];
b[i]=b[i+1];
b[i+1]=c;
}
}
}
for(int i=0;i<20;i++)
{
cout<<b[i]<<" ";
}
}

File: assignment1/question4/array.py
import random
import numpy as np
n=int(input('How many random numbers would you like? '))
def rand(n):
    a=[]
    for i in range(0,n):
        a.append(random.randint(1,1000))
    return a
print(rand(n))
a=rand(20)
print('20 random numbers are ')
print(a)    
print('20 random numbers in ascending order are ')
for i in range(0, 20):    
    for j in range(i+1, 20):    
        if(a[i] > a[j]):    
            b = a[i];    
            a[i] = a[j];    
            a[j] = b;
print(a)        
